From 9c8acb69771dfe0f893bdb887027567e80e6f518 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Benoit=20Gr=C3=A9goire?= <benoitg@coeus.ca>
Date: Tue, 5 Mar 2019 12:33:55 -0500
Subject: [PATCH] Make file format less stringent (match '<OFX' instead of
 '<OFX>' so that it will tolerate a default namespace declaration).  Doesn't
 help for non-default namespace usage such as
 https://bugs.gnucash.org/show_bug.cgi?id=797125

---
 lib/file_preproc.cpp | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/lib/file_preproc.cpp b/lib/file_preproc.cpp
index 68f063d..6ba607f 100644
--- a/lib/file_preproc.cpp
+++ b/lib/file_preproc.cpp
@@ -135,7 +135,7 @@ enum LibofxFileFormat libofx_detect_file_type(const char * p_filename)
           input_file.clear();
         }
 
-        if (s_buffer.find("<OFX>") != string::npos || s_buffer.find("<ofx>") != string::npos)
+        if (s_buffer.find("<OFX") != string::npos || s_buffer.find("<ofx") != string::npos)
         {
           message_out(DEBUG, "libofx_detect_file_type():<OFX> tag has been found");
           retval = OFX;
@@ -161,8 +161,3 @@ enum LibofxFileFormat libofx_detect_file_type(const char * p_filename)
     message_out(ERROR, "libofx_detect_file_type(): Failed to identify input file format");
   return retval;
 }
-
-
-
-
-
